src/business.ts

```diff
@@ -0,0 +1,121 @@
+import type { Marqeta, MarqetaOptions, MarqetaError } from './'
+
+export interface BusinessIdentifications {
+  type: string;
+  value: string;
+}
+
+export interface IncorporationType {
+  incorporationType: string;
+  stateOfIncorporation: string;
+}
+
+export interface Address {
+  address1: string;
+  address2: string;
+  city: string;
+  state: string;
+  postalCode: string;
+  country: string;
+}
```

Comment:
I can't believe all these are _required_ fields... are they? `address2` is **_required_**? My guess is that this is the complete set of fields, but not what you really want here.

Reply:
Right, address2 is not required.